docs/03consensus/proof-of-space.md

```diff
@@ -35,9 +35,11 @@ See our [plotting FAQ](https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#
 ## Farming
 Farming is the process by which a farmer receives a sequence of 256-bit challenges to prove that they have legitimately put aside a defined amount of storage. In response to each challenge, the farmer checks their plots, generates a proof, and submits any winning proofs to the network for verification.
 
-The process of inputting a challenge and outputting a proof involves multiple table lookups. First, the farmer responds to a challenge by reading a pair of values in table 7. These point to two entries in table 6, four entries in table 5, etc.
+  > Note: The next paragraph gives an estimate of the disk reads required to fetch a full proof. This actual result could be +/- two reads, depending on the checkpoint tables. 
 
-Finally, the farmer fetches the whole tree of x-values. This requires one disk read for table 7, two for table 6, four for table 5, etc. The whole process thus requires 64 disk reads, which takes approximately 640 ms on a slow HDD with a 10 ms seek time. The amount of data read is small and is independent of plot size.
+The process of inputting a challenge and outputting a proof involves multiple table lookups. First, the farmer reads the f7 value from table 7, the location of which is determined by reading two _C_ tables. After a matching f7 value is found, the farmer reads a single _line point_ in table 7, which contains two _back pointers_ to table 6, four to table 5, eight to table 4, sixteen to table 3, and thirty-two to table 2. This results in a total of 64 disk reads (1 f7 value and 63 line points) being performed to fetch the whole tree of 64 x-values. The 64 disk reads require approximately 640 ms on a slow HDD with a 10 ms seek time. The amount of data read is small and is independent of plot size.
```

Comment:
A few minor corrections here:
After the f7 is found, the value read from T7 is a single index of k+1 bits. These indices are serialized as "park 7" which are just bit-packed entries but not compressed as a line point. 

These P7 index is an index into Table 6, which does contain line points, as do the rest of the tables all the way back as you specified.